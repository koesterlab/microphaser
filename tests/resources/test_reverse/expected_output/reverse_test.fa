>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>5302a9716f887d3R
gccgggaccgggacAgggactggggcc
>c78a8dc5274f849R
ggggccgggaccgggacAgggactggg
>abb5a5249057784R
actggggccgggaccgggacAgggact
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
