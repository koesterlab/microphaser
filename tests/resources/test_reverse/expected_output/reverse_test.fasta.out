>62cb2647a87e5dfR
TGAGCCACACCCATCACTCAACATGTC
>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>6fb08ec7c7f47bcR
TTCTGAGCCACACCCATCACTCAACAT
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>694cc3c630d8067R
TTCTTCTGAGCCACACCCATCACTCAA
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>39562cfb831a701R
CTTTTCTTCTGAGCCACACCCATCACT
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>0d1064b66ef872eR
GATCTTTTCTTCTGAGCCACACCCATC
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>9dfd4a2a3e06a03R
TGAGATCTTTTCTTCTGAGCCACACCC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>1890ce6eb553e2eR
GACTGAGATCTTTTCTTCTGAGCCACA
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>bbcca5e5926c6b8R
AGGGACTGAGATCTTTTCTTCTGAGCC
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>4b3b6ad8c866d79R
GGAAGGGACTGAGATCTTTTCTTCTGA
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>c9376de4a4eeb36R
CAGCTCACTAAGATCCTGCAGGCTGGC
>1c060adfd690c42R
CAaCTCACTAAGATCCTGCAGGCTGGC
>70f4032847e8b9cR
CACCAGCTCACTAAGATCCTGCAGGCT
>8584b7aeb9d2f26R
CACCAaCTCACTAAGATCCTGCAGGCT
>89e88d210eb44afR
TACCACCAGCTCACTAAGATCCTGCAG
>a5528732feff657R
TACCACCAaCTCACTAAGATCCTGCAG
>5467b8035befff8R
CTCTACCACCAGCTCACTAAGATCCTG
>33bdff14891109fR
CTCTACCACCAaCTCACTAAGATCCTG
>20ad676ed0eeb1fR
ACCCTCTACCACCAGCTCACTAAGATC
>1e6e5cfb1cdc9deR
ACCCTCTACCACCAaCTCACTAAGATC
>63b7186258fc02dR
GTCACCCTCTACCACCAGCTCACTAAG
>86aae2a3ec5fcadR
GTCACCCTCTACCACCAaCTCACTAAG
>3f7096abd8bf649R
GGTGTCACCCTCTACCACCAGCTCACT
>caa2a4b00ce540eR
GGTGTCACCCTCTACCACCAaCTCACT
>caf61cdec0e89e5R
TTGGGTGTCACCCTCTACCACCAGCTC
>5aaa3507d1d1348R
TTGGGTGTCACCCTCTACCACCAaCTC
>36dee4a50c2493eR
CACTTGGGTGTCACCCTCTACCACCAG
>c204eec32d41624R
CACTTGGGTGTCACCCTCTACCACCAa
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>577e9ef1d73e376R
TGCCGGCGTGTTGGGTGACTCCTGGCT
>6744a2c2d41f673R
TGCCGGtGTGTTGGGTGACTCCTGGCT
>1e499d31afad30fR
CGGCGTGTTGGGTGACTCCTGGCTCCT
>a89c8ea6bd3c93eR
CGGtGTGTTGGGTGACTCCTGGCTCCT
>ffb22cb3519f46cR
CGTGTTGGGTGACTCCTGGCTCCTCTC
>dabf6a620bf78c7R
tGTGTTGGGTGACTCCTGGCTCCTCTC
>8e681aa30658d5aR
GGGTGCCGGCGTGTTGGGTGACTCCTG
>871c2db59be2e24R
GGGTGCCGGtGTGTTGGGTGACTCCTG
>fb16dc2abaad395R
CGGGGGTGCCGGCGTGTTGGGTGACTC
>a50701944aeb40fR
CGGGGGTGCCGGtGTGTTGGGTGACTC
>e263dcf1cffd1ecR
GAGCGGGGGTGCCGGCGTGTTGGGTGA
>6cbeb500d27ab0cR
GAGCGGGGGTGCCGGtGTGTTGGGTGA
>8870fc955acfa6aR
GGAGAGCGGGGGTGCCGGCGTGTTGGG
>9f825be514211d0R
GGAGAGCGGGGGTGCCGGtGTGTTGGG
>2a0306ddf027cf8R
CATGGAGAGCGGGGGTGCCGGCGTGTT
>380d4855b31900cR
CATGGAGAGCGGGGGTGCCGGtGTGTT
>8afcef1a5734f6aR
CAGCATGGAGAGCGGGGGTGCCGGCGT
>47689d704d16372R
CAGCATGGAGAGCGGGGGTGCCGGtGT
>e45b398cee8397eR
gccgggaccgggaccgggacagggacc
>aee7752445f7ac7R
gccgggaccgggacagggaccaggacT
>06e4d6fa2d0a92cR
ggggccgggaccgggaccgggacaggg
>e153e44397d0225R
ggggccgggaccgggacagggaccagg
>086ad8ff06c33deR
actggggccgggaccgggaccgggaca
>f645aea7b0c34e0R
acAggggccgggaccgggaccgggaca
>bc7fb48dd3846c6R
actggggccgggaccgggacagggacc
>0f1f6e85f76e173R
gggactggggccgggaccgggaccggg
>ce35a11bf24961fR
gggacAggggccgggaccgggaccggg
>7e52ccba441a7b0R
gggactggggccgggaccgggacaggg
>1e4997b22ab45d2R
accgggactggggccgggaccgggacc
>d5223fe536dff83R
accgggacAggggccgggaccgggacc
>0b4e37d09934bdeR
acAgggacAggggccgggaccgggacc
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>9be96014cc873e3R
gggaccgggactggggccgggaccggg
>138a4cf12fb852fR
gggaccgggacAggggccgggaccggg
>5829bfd284f0cb5R
gggacAgggacAggggccgggaccggg
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>5935263355d21baR
accgggaccgggactggggccgggacc
>d2e6552849f37efR
accgggaccgggacAggggccgggacc
>4087657e230f3a5R
accgggacAgggacAggggccgggacc
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>fd0f95bc205cdf2R
gggaccgggaccgggactggggccggg
>8c3434d759140efR
gggaccgggacAgggactggggccggg
>7397d811191e389R
gggaccgggaccgggacAggggccggg
>ae280e232d0582aR
gggaccgggacAgggacAggggccggg
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>f7a8162bb062070R
gccgggaccgggaccgggactggggcc
>561f80087272f47R
gccgggaccgggacAgggactggggcc
>d58bdfdd6863443R
gccgggaccgggaccgggacAggggcc
>e1b098d6a1588e2R
gccgggaccgggacAgggacAggggcc
>7fc6bfc369eeed6R
ggAgccgggaccgggaccgggactggg
>e9da669b9cab341R
ggAgccgggaccgggacAgggactggg
>81f77cbd10af71eR
ggggccgggaccgggaccgggacAggg
>4e047277cdeb322R
ggggccgggaccgggacAgggacAggg
>9e90dc0ffb54782R
actggAgccgggaccgggaccgggact
>fc553bd6250d877R
actggAgccgggaccgggacAgggact
>3e7efe637795baaR
actggggccgggaccgggaccgggacA
>f561469dc0d1bbdR
actggggccgggaccgggacAgggacA
>35401280f1deda7R
gggactggggccgggaccgggaccggg
>f810e9a2bcda534R
gggactggAgccgggaccgggaccggg
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>cf58ccde40a93c6R
gggactggAgccgggaccgggacAggg
>4918d5ffe004d7bR
acAgggactggggccgggaccgggacc
>e12e98c52f620f9R
accgggactggAgccgggaccgggacc
>64c8cd484eb231fR
acAgggactggAgccgggaccgggacc
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
>d6d3ec26afc54dbR
acAgggactggAgccgggaccgggacA
>560715910a2259fR
gggaccgggactggggccgggaccggg
>5391df73af8c82aR
gggacAgggactggggccgggaccggg
>39761e2062516f3R
gggaccgggactggAgccgggaccggg
>f4e8a163ae75493R
gggacAgggactggAgccgggaccggg
>f762b9aff44b7baR
accgggaccgggactggggccgggacc
>84d9899b511e010R
accgggacAgggactggggccgggacc
>2df06d869428c32R
accgggaccgggactggAgccgggacc
>dfcafdfb3ef2979R
accgggacAgggactggAgccgggacc
>ed4c5d4ddac6d98R
gggaccgggaccgggactggggccggg
>54a431782b5c55cR
gggaccgggacAgggactggggccggg
>fa985240c3ace48R
gggaccgggaccgggactggAgccggg
>b7f33ffb31bf247R
gggaccgggacAgggactggAgccggg
>076975fa347c701R
gccgggaccgggaccgggactggggcc
>a3bb17952d14d8eR
gccgggaccgggacAgggactggggcc
>41b133bbed1b4c3R
gccgggaccgggaccgggactggAgcc
>d370270b66cf113R
gccgggaccgggacAgggactggAgcc
>828e75a3e7deeeaR
ggggccgggaccgggaccgggactggg
>6d755f33d143c50R
ggggccgggaccgggacAgggactggg
>70c08db11e4a3dbR
ggggccgggaccgggaccgggactggA
>ea7d196af05dbe5R
ggggccgggaccgggacAgggactggA
>bdbf32be8271a0cR
actggggccgggaccgggaccgggact
>6b4e93ae8511e16R
actggggccgggaccgggacAgggact
>7b456d8781464bdR
gggactggggccgggaccgggaccggg
>cd6ce798f608164R
gggactggggccgggaccgggacAggg
>45531e958a7133eR
acagggactggggccgggaccgggacc
>02661a828cb1265R
acagggactggggccgggaccgggacA
